src/components/skeletons/SkeletonProductGrid/SkeletonProductGrid.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { ITEMS_PER_PAGE } from 'src/constants'
+
+import SkeletonProductCard from '../SkeletonProductCard'
+
+import 'src/components/product/ProductGrid/product-grid.scss'
+
+interface Props {
+  loading?: boolean
+  children?: JSX.Element
+}
+
+function SkeletonProductGrid({ children, loading = true }: Props) {
```

Comment:
```suggestion
function SkeletonProductGrid({ children, loading = true }: PropsWithChildren<Props>) {
```

Reply:
when I tried to use the type `PropsWithChildren` it raised an error when using the component as wrapper:

<img width="821" alt="Screen Shot 2022-02-16 at 23 13 15" src="https://user-images.githubusercontent.com/15722605/154391550-6e99d3a4-7f42-4b3c-a025-b84a5bbbea47.png">

that's why I was defining `children` as a JSX.Element. Do you have any thoughts about how we handle that?

Reply:
I solve that by wrapping `children` into a fragment since a component needs to return a single element.